src/main/java/org/myrobotlab/service/Log.java

```diff
@@ -25,23 +25,18 @@
 
 package org.myrobotlab.service;
 
-import java.lang.reflect.Method;
-import java.net.URI;
+import java.io.IOException;
 import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedList;
 import java.util.List;
 
-import org.myrobotlab.framework.MRLListener;
-import org.myrobotlab.framework.Message;
+import javax.annotation.processing.FilerException;

```

Comment:
Can we choose a different exception class there?  javax.annotation isn't super java11 friendly.

Reply:
ZOIKS !  - that was there for  testing purposes only - should have removed it  - will remove it

Reply:
removed